### PR TITLE
Create navbar

### PR DESCRIPTION
**Pull request by Robin-Git-Hub**
Finish fisrt version Navbar
Possibility to modify
Possibility of modifying if the visual is not suitable

### CODE REVIEW

File: app/assets/stylesheets/config/_fonts.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,100;1,400;1,500;1,700;1,900&display=swap');
 
 // Define fonts for body and headers
 $body-font: "Work Sans", "Helvetica", "sans-serif";
```
Comment (maxberko) [inline app/assets/stylesheets/config/_fonts.scss:5]: We should delete this line

File: app/assets/stylesheets/config/_fonts.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
```
Comment (maxberko) [inline app/assets/stylesheets/config/_fonts.scss]: We should delete this line, unless we're using this font ?

I thought we were using roboto

**Review by maxberko — COMMENTED** [review]
Je pense qu'il y a des items à changer au niveau du look&feel mais c'est pas grave

Sur la font je t'ai laissé deux comms

**Review by Romain999 — APPROVED** [review]
OK pour wam

File: app/assets/stylesheets/config/_fonts.scss (hunk 2 above)
Comment (Robin-Git-Hub) [reply app/assets/stylesheets/config/_fonts.scss]: I deleted this line. All that's left is to import roboto into the font config.

File: app/assets/stylesheets/config/_fonts.scss (hunk 1 above)
Comment (Robin-Git-Hub) [reply app/assets/stylesheets/config/_fonts.scss:5]: Do you also want to remove roboto import?

**Review by Manone22 — APPROVED** [review]

**Pull request merged**